docs/src/main/sphinx/release/release-381.md

```diff
@@ -0,0 +1,90 @@
+# Release 381 (11 May 2022)
+
+## General
+
+## Security
+
+* Add support for automatic discovery of OpenID Connect metadata with OAuth 2.0
+  authentication. ({issue}`9788`)
+* Re-introduce `ldap.ssl-trust-certificate` as legacy configuration to avoid
+  update failures. ({issue}`12187`)
+* Fix query failure when a table has multiple column masks defined. ({issue}`12262`)
```

Comment:
@kokosing here is the reworded entry .. please confirm

Reply:
👍

Reply:
It is not that it was not working at all. Usually it was working, but in some case it wasn't. I am not sure if I should go into details here.

Reply:
> Fix potential query failure when ...